### PR TITLE
Registration service documentation

### PR DESCRIPTION
**Pull request by dolgopolovwork**
Registration service documentation

### CODE REVIEW

File: btc-registration/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Bitcoin registration service
+The Bitcoin registration service (or simply `btc-registration`) is a service that responsible for assigning Bitcoin MultiSig addresses to Iroha clients.
```
Comment (Mingela) [inline btc-registration/README.md]:
```suggestion
The Bitcoin registration service (or simply `btc-registration`) is a service responsible for assigning Bitcoin MultiSig addresses to Iroha clients.
```

**Review by Mingela — APPROVED** [review]

File: btc-registration/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## Bitcoin registration service
+The Bitcoin registration service (or simply `btc-registration`) is a REST based service responsible for assigning Bitcoin MultiSig addresses to Iroha clients.
```
Comment (LiraLemur) [inline btc-registration/README.md:2]: Do you really want The before Bitcoin registration? REST-based

File: btc-registration/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+### Configuration overview (registration.properties)
+* `btc-registration.registrationAccount` - Iroha account that registers clients in D3
+* `btc-registration.mstRegistrationAccount` - Iroha account that creates Bitcoin MultiSig addresses
+* `btc-registration.freeAddressesStorageAccount` - Iroha account that stores free to register Bitcoin MultiSig addresses.
```
Comment (LiraLemur) [inline btc-registration/README.md:6]: addresses available for registration?

File: btc-registration/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+## Bitcoin registration service
+The Bitcoin registration service (or simply `btc-registration`) is a REST based service responsible for assigning Bitcoin MultiSig addresses to Iroha clients.
+### Configuration overview (registration.properties)
+* `btc-registration.registrationAccount` - Iroha account that registers clients in D3
+* `btc-registration.mstRegistrationAccount` - Iroha account that creates Bitcoin MultiSig addresses
+* `btc-registration.freeAddressesStorageAccount` - Iroha account that stores free to register Bitcoin MultiSig addresses. 
+* `btc-registration.nodeId` - identifier of the node. This identifier must correlate to an identifier that is set in `btc-address-generation` configuration file on the same node. This value must be different on different nodes.
```
Comment (LiraLemur) [inline btc-registration/README.md:7]: correlate with

File: btc-registration/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+* `btc-registration.irohaQueryTimeoutMls` - Iroha query timeout in milliseconds. We need this value to be set in order to improve the service liveness. The service re-reads requested data from Iroha if it fails to do so from the first attempt. If the service reaches the specified timeout, it returns an error.
```
Comment (LiraLemur) [inline btc-registration/README.md:8]: maybe "If the time is out, it returns an error"?

**Review by LiraLemur — COMMENTED** [review]
Please check

**Pull request merged**